### ISSUE TITLE
1ea59c ("video element visual content has audio description") should be regarded as required to WCAG 2.1 conformance

### ISSUE THREAD

**Issue by ebalink** [act-rules/act-rules.github.io#1709]
The accessibility requirements mapping in rule [1ea59c ](https://act-rules.github.io/rules/1ea59c) are missing a direct mapping to [AA-level WCAG 2,.1-rule 1.2.5.](https://www.w3.org/TR/WCAG21/#audio-description-prerecorded)
Because of this rule indicates that it isn't a requirement for WCXAG comformance even though it is.

For the situations described in [WCAG 2.1 1.2.3](https://www.w3.org/TR/WCAG21/#audio-description-or-media-alternative-prerecorded) one can choose between [1ea59 ](https://act-rules.github.io/rules/1ea59c)and [ee13b5](https://act-rules.github.io/rules/ee13b5).

**Comment by WilcoFiers** [comment]
See my response on https://github.com/act-rules/act-rules.github.io/issues/1710#issuecomment-919029530.
